test/augmentation/test_augmentation.py

```diff
@@ -10,10 +11,57 @@
 import kornia
 import kornia.testing as utils  # test utils
 from kornia.constants import pi
-from kornia.augmentation import RandomHorizontalFlip, RandomVerticalFlip, ColorJitter, \
+from kornia.augmentation import AugmentationBase, RandomHorizontalFlip, RandomVerticalFlip, ColorJitter, \
     RandomErasing, RandomGrayscale, RandomRotation, RandomCrop, RandomResizedCrop, RandomMotionBlur
 
 
+class TestAugmentationBase:
+
+    def test_forward(self, device):
+        torch.manual_seed(42)
+        input = torch.rand((2, 3, 4, 5)).to(device)
```

Comment:
not critical but it's recommended to initialize like this:
`input = torch.rand((2, 3, 4, 5), device=device, dtype=dtype)`